vtex-apps/my-account
# Fix logout translation

#### What is the purpose of this pull request?

The logout button and modal translations were not accurate.

#### How should this be manually tested?

Go to the following workspace, go to my account and click the logout button. Check that it now displays logout instead of "Leave" and in the modal instead of "quit".

https://arthur--recorrenciaqa.myvtex.com/account#/profile

#### Types of changes
